### DISCUSSION TITLE
Nova Cinema

### DISCUSSION THREAD

**Discussion by Mirekbfive** [iptv-org/community#874] [Q&A]
### Channel ID

NovaCinema.cz

### Stream URL

https://iptv-org.github.io/epg/guides/cs/mujtvprogram.cz.xml

### Notes

please edit the channel for the Fermata application, thank you

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by Mirekbfive** [comment]
https://iptv-org.github.io/epg/guides/cs/m.tv.sms.cz.xml
Nova , nova gold, nova fun, nova cinema,nova lady

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19834
